x-pack/plugins/cases/common/api/cases/configure.ts

```diff
@@ -9,18 +9,34 @@ import * as rt from 'io-ts';
 
 import { UserRT } from '../user';
 import { CaseConnectorRt, ConnectorMappingsRt, ESCaseConnector } from '../connectors';
+import { OmitProp } from '../runtime_types';
+import { OWNER_FIELD } from './constants';
 
 // TODO: we will need to add this type rt.literal('close-by-third-party')
 const ClosureTypeRT = rt.union([rt.literal('close-by-user'), rt.literal('close-by-pushing')]);
 
 const CasesConfigureBasicRt = rt.type({
+  /**
+   * The external connector
+   */
   connector: CaseConnectorRt,
+  /**
+   * Whether to close the case after it has been synced with the external system
+   */
   closure_type: ClosureTypeRT,
+  /**
+   * The plugin owner that manages this configuration
+   */
+  owner: rt.string,
```

Comment:
I am wondering if we can always have owner define like that `rt.union([rt.array(rt.string), rt.string])`

Reply:
Hey! It doesn’t make sense to have a `string[]` when you create a case because you can not assign multiple owners to a case. So instead of doing it like `const o = owner[0]` it is best to enforce on the schema level. Do we want to assign multiple owners on a case?

Reply:
I added it here to track the work: https://github.com/elastic/kibana/issues/100804

Xavier and I talked offline and we'll address this in a separate PR.